### DISCUSSION TITLE
Possibly incorrect checking of Firewall Notification settings - How to get the finding lists corrected?

### DISCUSSION THREAD

**Discussion by MrZuerrer** [0x6d69636b/windows_hardening#152] [Q&A] [answered]
The checks for the "DisableNotifications" setting in the firewall section seem to be reversed and thus incorrectly reporting inconsistencies as follows:

[😼] ID 10507, Display a notification (Domain Profile, Policy), Result=0, Recommended=1, Severity=Low
[😼] ID 10521, Display a notification (Private Profile, Policy), Result=0, Recommended=1, Severity=Low
[😼] ID 10535, Display a notification (Public Profile, Policy), Result=0, Recommended=1, Severity=Low

The checks are included in the following lists.

"finding_list_msft_security_baseline_windows_10_22h2_machine.csv"
"finding_list_msft_security_baseline_windows_10_2004_machine.csv"
"finding_list_msft_security_baseline_windows_11_21h2_machine.csv"
"finding_list_msft_security_baseline_windows_11_22h2_machine.csv"
"finding_list_bsi_sisyphus_windows_10_nd_machine.csv"
"finding_list_bsi_sisyphus_windows_10_ne_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_20h2_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_21h1_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_21h2_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_1809_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_1903_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_1909_machine.csv"
"finding_list_cis_microsoft_windows_10_enterprise_2004_machine.csv"
"finding_list_cis_microsoft_windows_11_enterprise_21h2_machine.csv"
"finding_list_cis_microsoft_windows_server_2012_r2_2.4.0_machine.csv"
"finding_list_cis_microsoft_windows_server_2016_1607_1.2.0_machine.csv"
"finding_list_cis_microsoft_windows_server_2016_1607_1.3.0_machine.csv"
"finding_list_cis_microsoft_windows_server_2019_1809_1.1.0_machine.csv"
"finding_list_cis_microsoft_windows_server_2019_1809_1.2.1_machine.csv"
"finding_list_cis_microsoft_windows_server_2022_21h1_1.0.0_machine.csv"
"finding_list_msft_security_baseline_windows_10_20h2_21h1_machine.csv"
"finding_list_msft_security_baseline_windows_10_21h2_machine.csv"

 I would like to understand (1) if the checks are indeed incorrectly set up and (2) how to get them corrected in the above "official" findings lists.

Many thanks for your support
Peter

**Comment by 0x6d69636b** [comment, marked as answer]
GPO text are sometimes unnecessary complicated, I use admx.help to check for settings:
https://admx.help/?Category=Windows_8.1_2012R2&Policy=Microsoft.Policies.WindowsFirewall::WF_Notifications_Name_1

If DisableNotifications is set to enabled (=1), Windows Firewall prevents the display of the notifications.

**Comment by MrZuerrer** [comment]
Thank you for the pointer.
I think I understand. It becomes a matter of personal preference in this particular case whether an administrator finds it more important to tell users (=0) or to not bother them (=1). I personally like to know.
